disco/preprocess/select_timepoints2.py

```diff
@@ -5,154 +5,310 @@
 @author: ksedzro
 """
 
-# -*- coding: utf-8 -*-
-"""
-Created on Fri Nov 25 07:43:46 2022
-
-@author: ksedzro
-"""
+import filecmp
+import enum
+import logging
+import shutil
+import time
+from pathlib import Path
 
-import os
 import numpy as np
 import pandas as pd
-import time
 import opendssdirect as dss
 
-def load_feeder(path_to_master):
-    if os.path.exists(path_to_master):
-        dpath, fname = os.path.split(path_to_master)
-        newfname = fname.split('.')[0]
-        new_master = os.path.join(dpath, f"{newfname}_new.dss")
-        with open(path_to_master, 'r') as mr:
-            slines = mr.readlines()
-        lines = []
-        bk = False
-        for line in slines:
-            
-            if line.lower().startswith('solve'):
-                line = "Solve\n"
-                bk = True
-            lines.append(line)
-            if bk:
-                break
-        with open(new_master, 'w') as mw:
-            mw.writelines(lines)
-            
-    r = dss.run_command(f"redirect {new_master}")
-    if r != '':
-        print(f"PoenDSS error: {r}")
-        
-    
-def get_param_values(param_class, bus_data, category='demand'):
-    assert category in ['demand', 'generation']
-    
+
+logger = logging.getLogger(__name__)
+
+
+_DISALLOWED_OPEN_DSS_COMMANDS = ("export", "plot", "show")
```

Comment:
Do we need to disallow various types of `set mode` commands? If so, what is the complete list?